src/services/AbstractDFG.jl

```diff
@@ -432,6 +432,82 @@ function _copyIntoGraph!(sourceDFG::G, destDFG::H, variableFactorLabels::Vector{
     return nothing
 end
 
+#TODO UNTESTED and NOT FILLED IN, just to define function signatures
+"""
+    $(SIGNATURES)
+"""
+function getVariableSolverData(dfg::AbstractDFG, variablekey::Symbol, solvekey::Symbol=:default)
+    error("not implemented")
+    return deepcopy(solverData(getVariable(dfg, variablekey), solvekey))
```

Comment:
why a deepcopy?

Reply:
should verb `get` not return the original memory object?

Reply:
Yes, I think you are correct.